fsw/cfe-core/src/tbl/cfe_tbl_api.c

```diff
@@ -678,21 +678,13 @@ int32 CFE_TBL_Load( CFE_TBL_Handle_t TblHandle,
     int32                       Status;
     uint32                      ThisAppId;
     CFE_TBL_LoadBuff_t         *WorkingBufferPtr;
-    CFE_TBL_AccessDescriptor_t *AccessDescPtr = &CFE_TBL_TaskData.Handles[TblHandle];
-    CFE_TBL_RegistryRec_t      *RegRecPtr = &CFE_TBL_TaskData.Registry[AccessDescPtr->RegIndex];
-    char                        AppName[OS_MAX_API_NAME]={"UNKNOWN"};
+    CFE_TBL_AccessDescriptor_t *AccessDescPtr;
+    CFE_TBL_RegistryRec_t      *RegRecPtr;
+    char                        AppName[OS_MAX_API_NAME] = {"UNKNOWN"};
     bool                        FirstTime = false;
 
     /* Verify access rights and get a valid Application ID for calling App */
-    Status = CFE_TBL_ValidateAccess(TblHandle, &ThisAppId);
-
-    /* Translate AppID of caller into App Name */
-    CFE_ES_GetAppName(AppName, ThisAppId, OS_MAX_API_NAME);
-
-    /* Initialize return pointer to NULL */
-    WorkingBufferPtr = NULL;
-
-    if (Status != CFE_SUCCESS)
+    if ((Status = CFE_TBL_ValidateAccess(TblHandle, &ThisAppId)) != CFE_SUCCESS)
```

Comment:
I know I'm a pain (and feel free to ignore this) but isn't assignments within an "if" statement also a no-no per coding standards?  I thought there was a preference to keep assignments and conditionals separated.

Reply:
Yup, section 3.10 of the C_Coding_Standard_20051129 that we've never seemed to have shared for some reason states: "Side effects within expressions should not be used."

Reply:
While I agree that side effects within expressions should be excluded, I feel this form is easy to understand, is a very common pattern in our code, and it is clear there's a side effect in the expression. I'm interested in hearing what the consensus of the CCB is with regards to this.

Sorry for the multiple pushes. I'll make the change now.